test/views/helpers/ResultsPageHelperSpec.scala

```diff
@@ -42,357 +44,487 @@ class ResultsPageHelperSpec extends SpecBase {
       "should return valid summary" in {
         val calculatorResult = SingleResult(MarginalRate(1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1))
 
-        displayYourDetails(
-          calculatorResult,
-          AccountingPeriodForm(
-            epoch,
-            Some(epoch.plusDays(1))
-          ),
-          1,
-          11,
-          111,
-          true
-        ).body.trimNewLines mustBe
-          """<h2 class="govuk-heading-m" style="margin-bottom: 4px">resultsPage.yourDetails</h2>
-            | <dl class="govuk-summary-list govuk-summary-list--no-border">
-            |   <div class="govuk-summary-list__row">
-            |     <dt class="govuk-summary-list__key">resultsPage.accountPeriod</dt>
-            |     <dd class="govuk-summary-list__value">site.from.to</dd>
-            |   </div>
-            |   <div class="govuk-summary-list__row">
-            |     <dt class="govuk-summary-list__key">resultsPage.companysProfit</dt>
-            |     <dd class="govuk-summary-list__value">£1</dd>
-            |   </div>
-            |   <div class="govuk-summary-list__row">
-            |     <dt class="govuk-summary-list__key">resultsPage.distributions</dt>
-            |     <dd class="govuk-summary-list__value">£11</dd>
-            |   </div>
-            |   <div class="govuk-summary-list__row">
-            |     <dt class="govuk-summary-list__key">resultsPage.associatedCompanies</dt>
-            |     <dd class="govuk-summary-list__value">111</dd>
-            |   </div>
-            | </dl>
-            | <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible"/>""".stripMargin.trimNewLines
+        Jsoup
+          .parse(
```

Comment:
as `Jsoup.parse("").body.html` is used in many places, does it make sense to add an implicit class for it?

```result.parseHtml shouldBe expected.parseHtml ```